vitrina/structure/models.py

```diff
@@ -75,6 +76,19 @@ class Metadata(models.Model):
     access = models.IntegerField(
         _("Prieiga"), choices=ACCESS_TYPES, blank=True, null=True
     )
+    visibility = models.ForeignKey(
+        Visibility,
+        models.SET_NULL,
+        verbose_name=_("Metaduomenų matomumas"),
+        null=True,
+        blank=True,
+    )
+    eli = models.URLField(
+        _("ELI identifikatorius"), blank=True, null=True, max_length=500
```

Comment:
Let's use an understandable verbose_name + the abbreviation in the brackets:

```suggestion
        _("Europos teisės aktų identifikatorius (ELI)"), blank=True, null=True, max_length=500
```

- https://eur-lex.europa.eu/eli-register/about.html?locale=lt